<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestTextCanvasMainDialog</class>
 <widget class="QDialog" name="QtTestTextCanvasMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>585</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #808080, stop: 1 #ffffff);
}
QPushButton { background-color: rgb(255, 255, 255);  }
QPushButton { color: rgb(0,0,0); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layout_dot">
     <item>
      <widget class="QPushButton" name="button_char">
       <property name="text">
        <string>Set char</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_char_x">
       <property name="text">
        <string>(x =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_char_x">
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_char_y">
       <property name="text">
        <string>, y=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_char_y">
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_char_close">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_char">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="char_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_circle">
     <item>
      <widget class="QPushButton" name="button_text">
       <property name="text">
        <string>Set text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_text_x">
       <property name="text">
        <string>(x =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_text_x">
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_text_y">
       <property name="text">
        <string>, y=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_text_y">
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_text_close">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_text">
       <property name="text">
        <string>Hello world</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="text_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_coordinat_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <property name="text">
        <string>Coordinat system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="box_coordinat_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>screen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>graph</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_clear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_as_dialog">
     <property name="text">
      <string>Show as dialog</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
